### PR TITLE
Pincode implementation

### PR DESCRIPTION
**Pull request by rip32700**

### CODE REVIEW

**Comment by rip32700** [comment]
Fixes #12

**Comment by maratsubkhankulov** [comment]
Looking good, but lets not ask user to input pin on disable.

**Comment by michaelScoff** [comment]
@maratsubkhankulov @magicaltramp why? You need to enter passcode to disable it

**Comment by rip32700** [comment]
@michaelScoff Marat thought it would be better without asking passcode to disable it since it is done like this in the iOS version and thus for consistency. I personally think it is a good idea to ask for it when disabling it, what do you think?

File: app/src/main/java/com/wallet/crypto/trustapp/views/CustomPinActivity.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+    @Override
+    public void showForgotDialog() {
+        AlertDialog.Builder builder;
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            builder = new AlertDialog.Builder(this, android.R.style.Theme_Material_Dialog_Alert);
+        } else {
+            builder = new AlertDialog.Builder(this);
+        }
+        builder.setTitle("Warning")
```
Comment (michaelScoff) [inline app/src/main/java/com/wallet/crypto/trustapp/views/CustomPinActivity.java]: Why do we need to contact devs? I don’t think we can help here 

Also strings should be added always to the special file, like android devs do

**Comment by michaelScoff** [comment]
Actually you are right, let’s keep it simple for now

**Comment by rip32700** [comment]
@maratsubkhankulov I changed to not ask for pincode when disabling
@michaelScoff I removed forgot dialog

@maratsubkhankulov @madcake It should work now with the latest master. Please review and test.
@madcake can you test fingerprint on a real device?

**Comment by maratsubkhankulov** [comment]
@rip32700 thanks Phil.

**Pull request merged**